packages/klevu-ui/src/components/klevu-chat-bubble/klevu-chat-bubble.css

```diff
@@ -11,6 +11,8 @@
   --background-remote: var(--klevu-chat-bubble-background-remote, var(--klevu-color-primary));
   --text-color: var(--klevu-chat-bubble-text-color, inherit);
   --text-color-remote: var(--klevu-chat-bubble-text-color-remote, var(--klevu-color-neutral-1));
+  --border-radius-general: var(--klevu-chat-bubble-border-radius-general, var(--klevu-border-radius-m));
+  --border-radius-corner: var(--klevu-chat-bubble-border-radius-general, 0); 
```

Comment:
do we need 2? the user can provide 4 diff values if they want diff radius for each corner?